lib/libc/math/lib_erfc.c

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ * lib/math/lib_erfc.c
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <math.h>
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+#ifdef CONFIG_HAVE_DOUBLE
```

Comment:
This file needs to include <tinyara/config.h> for this line.
What I left before does not have `#ifdef CONFIG_XX`. That's why I said they are unnecessary.

Reply:
CONFIG_HAVE_DOUBLE kinds of configs are in compiler.h. Let's keep original including headers.
I mean,
if there are `#ifdef config_have_double`, keep original including.
if there is no one, remove.

Reply:
Okay, done.